package.json

```diff
@@ -94,7 +95,7 @@
     "@types/smoothscroll-polyfill": "^0.3.1",
     "cypress": "^8.7.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "faker": "^5.5.3"
+    "faker": "5.5.3"
```

Comment:
Note: Pinning faker for safety
https://www.bleepingcomputer.com/news/security/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/